src/main/java/Main.java

```diff
@@ -0,0 +1,13 @@
+import core.basesyntax.Figure;
+import core.basesyntax.FigureSupplier;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[5];
+        FigureSupplier supplier = new FigureSupplier();
+        for (int i = 0; i < figures.length; i++) {
+            supplier.getRandomFigure();
+        }
+        Figure defaultFigure = supplier.getDefaultFigure();

```

Comment:
the task was to create 3 random figures and 3 default figures, call draw() for each of them here